### PR TITLE
[improvement](http) Session timeout is configurable, the default is 120 minutes

### PR DESCRIPTION
**Pull request by hf200012**
Session timeout is configurable, the default is 120 minutes

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No need)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -360,6 +360,12 @@
     @ConfField public static int jetty_server_acceptors = 2;
     @ConfField public static int jetty_server_selectors = 4;
     @ConfField public static int jetty_server_workers = 0;
+
+    /**
+     * session timeout，Unit: minutes
+     */
+    @ConfField public static int jetty_session_timeout = 120;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:367]:
```suggestion
    @ConfField public static int jetty_session_timeout_minute = 120;
```

**Review by morningman — COMMENTED** [review]
Please complete the PR template

**Comment by hf200012** [comment]
> Please complete the PR template

OK

**Comment by morningman** [comment]
Compile Error:
```
[ERROR] /root/doris/fe/fe-core/src/main/java/org/apache/doris/httpv2/HttpAuthManager.java:[38,53] cannot find symbol
[00:05:18](http://106.13.123.159:8111/buildConfiguration/IncubatorDoris_Build/3388?showLog=3388_13936_13936&logFilter=debug)
    symbol:   variable jetty_session_timeout
[00:05:18](http://106.13.123.159:8111/buildConfiguration/IncubatorDoris_Build/3388?showLog=3388_13937_13937&logFilter=debug)
    location: class org.apache.doris.common.Config
```

**Comment by hf200012** [comment]
> Compile Error:
> 
> ```
> 
> [ERROR] /root/doris/fe/fe-core/src/main/java/org/apache/doris/httpv2/HttpAuthManager.java:[38,53] cannot find symbol
> 
> [00:05:18](http://106.13.123.159:8111/buildConfiguration/IncubatorDoris_Build/3388?showLog=3388_13936_13936&logFilter=debug)
> 
>     symbol:   variable jetty_session_timeout
> 
> [00:05:18](http://106.13.123.159:8111/buildConfiguration/IncubatorDoris_Build/3388?showLog=3388_13937_13937&logFilter=debug)
> 
>     location: class org.apache.doris.common.Config
> 
> ```

Resolved

**Pull request closed without merging**